proposal/distributed-workloads.md

```diff
@@ -0,0 +1,79 @@
+
+*InstaScale*
+- Cluster elasticity catering for large distributed job requirements, realizing instant scaling by acquiring resources in the cloud in a demand-driven way based on queued jobs’ resource requests
+- Resources are acquired from the cloud provider at job start/finish time, and released immediately as soon as the job completes, which minimizes the cost of resources, as none are kept idle
+- Selects right provider instance sizes on behalf of the user
+
+Current state of community landscape regarding batch processing and scaling is summarized here:
+[Openshift Batch and Open Source landscape](https://docs.google.com/document/d/1N6cqIwylnDRtqHoxiq8GdwO7AbvkcNEdpWil2CzkzNU/edit?usp=sharing)
+
+**User Use Cases**
+- *Data Scientists* need self-service access to ephemeral high-performance compute infrastructure (i.e, multi-node GPU enabled clusters) for performing distributed computation. This platform supports both on-prem and cloud based environments.
+- *Cluster Administrators* can provide the capacity for on-demand infrastructure to their data scientist users, while implementing per user quota policies for fair access and limiting cloud spend. 
+- *Cluster Users’* requests of infrastructure for distributed compute jobs or interactive development environments will be placed in a queue until the necessary cluster resources become available.      
+
+## Architecture
+The architecture and typical workflow for a sample Ray distributed workload are depicted below.
+![Screenshot from 2022-12-02 14-57-05](https://user-images.githubusercontent.com/43946617/206355518-4b3ecd06-7290-4c4c-9f21-8bfb02011927.png)
+
+------------------------------------------------------------------------------------------------------------------------------
+
+![Screenshot from 2022-12-02 14-56-13](https://user-images.githubusercontent.com/43946617/206355725-96eba2dd-1b2f-489e-a396-981bdc6f0819.png)
+
+## Implementation
+The implementation of Distributed Workloads with Ray is demonstrated as part of the CodeFlare PoC demo:
+[Link to the demo](https://drive.google.com/file/d/1SxySCzxrJi1NZFf_kRlgoiV9ipWmXbhF/view?usp=sharing)
+
+## Owners Information and Maintenance Plan
+Selbi Nuryyeva - RHODS team - Red Hat
+
+Mustafa Eyceoz - PSAP team - Red Hat
+
+Atin Sood - IBM Research team - IBM
+
+The owners are committed to maintaining and testing the Distributed Workloads with each new release of its every component and ODH.
+
+## More Info
+CodeFlare SDK: https://github.com/project-codeflare/codeflare-sdk
+
+MCAD: https://github.com/IBM/multi-cluster-app-dispatcher → will be moving under CodeFlare organization Dec 2022
+
+InstaScale: https://github.com/IBM/multi-cluster-app-dispatcher/tree/cluster-autoscale  → will be moving under CodeFlare organization Dec 2022
```

Comment:
You can update this now that they have been relocated

Reply:
All links have been updated in a commit